sonaranalyzer-dotnet/its/expected/Ember-MM/generic.EmberCore.WebServer-{64D6F035-F186-4193-AF35-D20126B8DA64}-S3923.json

```diff
@@ -2,14 +2,14 @@
 "issues":  [
 {
 "id":  "S3923",
-"message":  "Remove this 'If' or edit its blocks so that they are not all the same.",
+"message":  "Remove this conditional structure or edit its code blocks so that they're not all the same.",
 "location":  {
 "uri":  "sources\Ember-MM\Addons\generic.EmberCore.WebServer\Server\clsServer.vb",
 "region":  {
 "startLine":  204,
 "startColumn":  17,
-"endLine":  206,
```

Comment:
@christophe-zurn-sonarsource will this re-create issues because of the change of location? probably yes, but that's life

Reply:
@andrei-epure-sonarsource . Just tested locally, and the tracking of `won't fix` issues still works. I think that the fact that the start line didn't change is good enough